src/views/Timeline.vue

```diff
@@ -464,7 +464,7 @@ export default {
       if (inf.length > 0) {
         topLabel = inf[0].label
       }
-      let icon = 'mdi-crosshairs-question'
+      let icon = 'mdi-karate'
```

Comment:
@vickywane here is a fall icon:

https://iconify.design/icon-sets/tabler/fall.html